packages/task/src/common/task-protocol.ts

```diff
@@ -97,6 +97,8 @@ export interface TaskExitedEvent {
     // Exactly one of code and signal will be set.
     readonly code?: number;
     readonly signal?: string;
+
+    readonly config?: TaskConfiguration;
```

Comment:
i know that having an optional property does not break API. but is it good to make `config` optional? all exited tasks have a `TaskConfiguration`, so it does not look optional conceptually. what do you think @akosyakov  ?